packages/slice-machine/server/src/api/tracking.ts

```diff
@@ -37,14 +46,23 @@ export function sendEvents(
       });
     }
   } else if (isTrackingEvent(event)) {
+    const maybeNodeVersion =
+      event.name === EventNames.PageView ? { nodeVersion } : {};
```

Comment:
😔 sad that we have to do it that way, but i understand why.
Could you just add a comment. to explain
- why we chose this events
- what kind of extra properties would be tracked in this one ?

Reply:
Also
- Why do we do that back-end and not frontend, why here ?